ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -57,14 +57,32 @@ class cls.
     pivotParameterTargetInput = InputData.parameterInputFactory("pivotParameterTarget", contentType=InputTypes.StringType,
                                                                 descr="""Pivot parameter for target inputs""")
     inputSpecification.addSub(pivotParameterTargetInput)
+    separateFeatureDataInput = InputData.parameterInputFactory("separateFeatureData", contentType=InputTypes.StringType,
+                                                                descr="""Time points to separate feature data and conduct DSS postprocessing
+                                                                on selected time interval. Values must be fractions of the full time interval.
+                                                                To distinguish strart and end, '|' should be placed between both values. Example: 0.0|0.4
+                                                                To provide more than one selection for the same data set,
+                                                                '+' should be placed at the end of each interval to start defining the new interval. Example: 0.0|0.4+0.4|1.0
+                                                                If 'None' is provided, the DSS postprocessing will be applied to the full interval.""")
+    separateFeatureDataInput.addParam("type", InputTypes.StringType)
```

Comment:
please add description for "type" using "descr"

Reply:
Description has been added for attribute "type" in separateFeatureData and separateTargetData.